### ISSUE TITLE
Guide is not completed. IoT Hub Connection issue

### ISSUE THREAD

**Issue by davlind87** [Azure/iothub-explorer#29]
Hi, 

Im following this guide https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-store-data-in-azure-table-storage and now I'm trying to save all of my messages. However, step 8 under "Get the connection string of your IoT hub endpoint" is getting on my nervs. 

I cannot find a way to "Create the connection string of your IoT hub endpoint as follows: Endpoint=<Event Hub-compatible endpoint>;SharedAccessKeyName=iothubowner;SharedAccessKey=<Primary key".

Any help is appreciated.

**Comment by pierreca** [comment]
Let's see if @dominicbetts can help. This the repository for iothub-explorer though, not for the docs. I believe this question would be better asked in the comments section of the page you're linking to.

**Comment by dominicbetts** [comment]
Hi @davlind87 - the connection string is a text value that you'll use later when you create the function app. Step 8 is asking you to make a note of the **Event Hub-compatible endpoint** and **Primary key** values displayed on the screen to construct the connection-string. I'll ask the author of the article to clarify this step.
If you have questions about the content of the articles on docs.microsoft.com, it's best to ask them in the comments section at the end of each article.

**Comment by shizn** [comment]
Hi @davlind87 . Per Dominic's instructions, the "create" actually means "construct" here. Later I will update the content to rephrase it to avoid confusion.
BTW, this issue is not related to iothub-explorer.

**Comment by davlind87** [comment]
Thanks for all of your support (community). Sorry for addressing the issue in the wrong thread though.  
@dominicbetts Your clarification made sense and what I needed to continue. Much appreciated.
